### PR TITLE
Tweaks to cookie and accessibility pages.

### PR DESCRIPTION
**Pull request by simonnebMOJ**
Minor tweaks to bring them in line with our other public services.

### CODE REVIEW

File: server/views/pages/cookies/cookies-policy.njk [hunk 1]
```diff
@@ -95,6 +95,38 @@
         <li>what you click on while you’re using the service</li>
       </ul>
 
+        <p class="govuk-body">Google Analytics sets the following cookies:</p>
+      {% from "govuk/components/table/macro.njk" import govukTable %}
```
Comment (psoleckimoj) [inline server/views/pages/cookies/cookies-policy.njk:99]: Generally better at the top of the page so you can see imports more easily.

File: server/views/pages/cookies/cookies-policy.njk [hunk 2]
```diff
@@ -95,6 +95,38 @@
...
+          ],
+          [
+            { text: "_gid" },
+            { text: "Checks if you’ve signed in to the service before. This helps us count how many people use the service." },
+            { text: "24 hours" }
+          ]
+        ]
+      }) }}
+
       {% from "govuk/components/radios/macro.njk" import govukRadios %}
```
Comment (psoleckimoj) [inline server/views/pages/cookies/cookies-policy.njk:130]: I realise these are here but should be at the top 😁

File: server/views/pages/cookies/cookies-policy.njk (hunk 1 above)
Comment (simonnebMOJ) [reply server/views/pages/cookies/cookies-policy.njk]: Moved to top

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**